src/js/view/controls/settings-menu.js

```diff
@@ -215,7 +215,7 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
     // Visual Quality
     model.on('change:visualQuality', (changedModel, quality) => {
         const qualitySubMenu = settingsMenu.getSubmenu('quality');
-        if (qualitySubMenu) {
+        if (quality && qualitySubMenu) {
```

Comment:
Can we trust that `quality.level` will always be defined? There's no guard in [`changeAutoLabel`](https://github.com/jwplayer/jwplayer/blob/f041d964b3e31ea32ffd4ad08359b0702bf9cac6/src/js/view/controls/settings-menu.js#L151) if it isn't and the function references `level.index`.